src/modules/85-iacm/components/IACMResourceStackWizard/StackRepoDetailsStep.tsx

```diff
@@ -12,6 +12,11 @@ import { useStrings } from 'framework/strings'
 import type { StackWizardStepProps } from './index'
 import css from './StackWizard.module.scss'
 
+export enum GitFetchTypes {
+  Branch = 'branch',
+  Commit = 'commitId'
```

Comment:
*SUPER minor and totally ignorable*, but a good convention is to use all caps for enums
```suggestion
  BRANCH = 'branch',
  COMMIT = 'commitId'
```